gcs-sink-connector/src/main/java/io/aiven/kafka/connect/gcs/GcsSinkConfig.java

```diff
@@ -438,4 +443,15 @@ public String getGcsEndpoint() {
     public String getUserAgent() {
         return getString(GCS_USER_AGENT);
     }
+
+    public ApiTracerFactory getApiTracerFactory() {
+        if (getClass(GCS_METRICS) == null) {
+            return null;
+        }
+        try {
+            return getClass(GCS_METRICS).asSubclass(ApiTracerFactory.class).getDeclaredConstructor().newInstance();
+        } catch (ReflectiveOperationException e) {
+            throw new RuntimeException(e);
```

Comment:
Can it be `ConfigException` with the message clarifying why the instantiation of `ApiTracerFactory` failed and which class is expected there?

Reply:
Done. Just my lazyness - its just that ConfigException doesn't follow normal exception chaining patterns